bootstrap-app/src/cmr/bootstrap/services/dispatch/dispatch_protocol.clj

```diff
@@ -0,0 +1,43 @@
+(ns cmr.bootstrap.services.dispatch.dispatch-protocol
```

Comment:
As noted above, I'd rename this to simply `dispatch.core` and move the `extend`s and implementation-agnostic functions here.